code-challenges.js

```diff
@@ -10,36 +10,78 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a conditional statement that evaluates two strings and determines which of the strings has more characters. Use the two sets of test variables provided.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
 // Expected outcome: "banana"
 
 // Set two:
 // const fruit1 = "cherry"
 // const fruit2 = "kiwi"
-// Expected outcome: "cherry"
+// // Expected outcome: "cherry"
 
 // Pseudo code:
+// I need to create a conditional statement (if/else) 
+// It needs to state that if the length of fruit1 is greater than fruit 2 then output fruit 1.
+// HOWEVER, if fruit 1 is not greater than fruit 2, output fruit 2 instead. 
+
+// if(fruit1.length > fruit2.length){
+//     console.log(fruit1)
+// }   else console.log(fruit2)
 
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// Pseudo code:
+// Pseudo code: 
+// I need to combine the two arrays which are [2, 5, 2, 2, 4] and [6,, 3, 5, 3] into one array.
+// Once the two arrays are combined, I need to run a .length to count the elements/items in the final array. 
+// The output will come out as 9 since there are 9 elements/items in the whole array now.
+// The below is the process into one whole line:
+
+// console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length) // --> 9
+
+// // The below is the process using a new variable to show the outcome of the combined two original variables's length.
```

Comment:
Here is how you could rewrite this pseudo code to be more concise:
- combine the given arrays using .concat
- log the new length of the concatenated array using .length

This explains exactly what the code is doing and keeps it short and sweet!